.github/workflows/release.yml

```diff
@@ -26,9 +26,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+        with:
+            fetch-depth: 0
       - uses: actions/setup-node@v3
         with:
           node-version: 16
       - run: npx changelogithub
```

Comment:
I saw this action was added in https://github.com/gosling-lang/gosling.js/pull/881. This action also won't work unless you set `fetch-depth: 0`.

Reply:
Ah, got it! Thanks for finding this